Makefile

```diff
@@ -89,7 +89,7 @@ BOOTSTRAP_ARGS?=
 BUILD_TAGS?=$(TAGS)
 WITH_LXD?=true
 WITH_OPENCONTRAIL?=true
-WITH_LIBVIRT?=true
+WITH_LIBVIRT_GO?=true
```

Comment:
why does it matter how its' implemented? rather leave the flag name `WITH_LIBVIRT`

Reply:
With this PR, we will be able to always enable the `libvirt` probe. WITH_LIBVIRT_GO will specify if we want to use the `libvirt-go` library (that includes a dynamic dependency on `libvirt.so`). When set to false, the pure Go implementation will be used. It has deployment implication so I think it makes sense to define a variable for it

Reply:
For the name of the variable itself, reusing WITH_LIBVIRT may let the user think that setting it to false will disable the `libvirt` probe.